data/player_data.go

```
@@ -41,5 +39,5 @@
 
 	DB = db
 
-	DB.AutoMigrate(&models.Player{})
+	DB.AutoMigrate(&model.Player{})
```

golangci-lint (1.64.8)

Error return value of `DB.AutoMigrate` is not checked

(errcheck)